basic-rec-track.svg

```diff
@@ -116,7 +116,7 @@
 								</g>
 								<text font-size="8">
 									<tspan x="182" y="112">WG Decision</tspan>
```

Comment:
This label overlaps exactly (and is redundant with) the one from the `<!-- CR to CRD -->` block.

Reply:
Would you say one should be deleted (and if so, which), or the redundant overlay should be retained?

Reply:
Addressed in https://github.com/w3c/w3process/pull/667/commits/d1bd24fa7e625a3b41799d183094133b2e9e607a